packages/hash/api/src/db/adapter.ts

```diff
@@ -157,6 +157,26 @@ export type Graph = {
   }[];
 };
 
+export type DbBlockProperties = {
+  entityId: string;
+  accountId: string;
+  componentId?: string;
+};
+
+export type DbPageProperties = {
+  archived?: boolean | null;
+  contents: DbBlockProperties[];
+  summary?: string | null;
+  title: string;
+};
+
+export type DbPage = {
+  type: "Page";
+  properties: DbPageProperties;
+};
```

Comment:
These specific db records are basically the same as every other entity except we know what the contents of their `properties` field are - so I suggest this is instead something like the following (renamed to `DbPageEntity` for consistency with the other DbXEntities)
```suggestion
export type DbPageEntity = Omit<DbEntity, "properties"> & {
  properties: DbPageProperties;
};
```